qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -675,6 +675,76 @@ def test_df_for_valid_file(self):
         self.validate_df("dumpfile")
 
 
+class TestQuota(TestCephFSShell):
+    dir_name = 'testdir'
+
+    def create_dir(self):
+        mount_output = self.get_cephfs_shell_cmd_output('mkdir ' + self.dir_name)
+        log.info("cephfs-shell mount output:\n{}".format(mount_output))
+
+    def validate(self, input_val):
+        quota_output = self.get_cephfs_shell_cmd_output('quota set --max_bytes '+input_val[0]+' --max_files '+input_val[1]+ ' '+ self.dir_name)
```

Comment:
nit: spaces before and after `+` would be nicer from readability POV

Reply:
Done.